eclass/kernel-2.eclass

```diff
@@ -600,10 +600,8 @@ kernel_is_2_6() {
 
 # Capture the sources type and set DEPENDs
 if [[ ${ETYPE} == sources ]]; then
-	DEPEND="!build? (
-		sys-apps/sed
-		>=sys-devel/binutils-2.11.90.0.31
-	)"
+	[[ ${EAPI} == [0-6] ]] && DEPEND="!build? ( sys-apps/sed )" ||
```

Comment:
We don't tend to use ranges for EAPI, as there is no guarantee (PMS) that it is, or will continue to be, numeric.

Reply:
In this case, it’s okay because we die on unknown EAPIs above anyway, and this is only a compatibility hack for older ones

Reply:
Hmmkay!